datadotworld/client/api.py

```diff
@@ -190,15 +180,6 @@ def update_dataset(self, dataset_key, **kwargs):
         """
         request = self.__build_dataset_obj(
             lambda: _swagger.DatasetPatchRequest(),
```

Comment:
Does the DatasetPatchRequest model need updating? Same question for the other dataset requests models.

Reply:
I'm assuming _all_ of the models need updating; if this one is in-scope, I'll update it

Reply:
They probably do, but like you said, we're only concerned with the models in scope of this ticket, which should be limited to the "Dataset*Request" and "Project*Request" models.